internal/compose/convert.go

```diff
@@ -52,6 +53,11 @@ func ConvertSpec(spec *score.WorkloadSpec) (*compose.Project, ExternalVariables,
 				})
 			}
 		}
+		// NOTE: Sorting is necessary for DeepEqual call within our Unit Tests to work reliably
```

Comment:
Should we be changing the code to make the tests work?
Doesn't this also have the advantage of making the generation of the compose file consistent (i.e. between runs with not changes in different environments the file will be the same?)

Reply:
This is a non-functional change. There is no need or requirement at this stage for the output arrays to be in a particular order. Thus is the comment, otherwise people would try to remove unnecessary sort in the future.